config/clusters/2i2c/ohw.values.yaml

```diff
@@ -80,10 +80,10 @@ basehub:
           authenticator_class: github
         GitHubOAuthenticator:
           oauth_callback_url: https://ohw.pilot.2i2c.cloud/hub/oauth_callback
-          allowed_organizations:
+          allowed_teams:
```

Comment:
I'm a bit confused 😬 Should I have allowed the entire org?

Reply:
Hmm. I also realized now that the teams are private. This shouldn't be an issue if they authorize the GitHub app. Righ, @sgibson91 ?

Reply:
No no no, I just mean the endpoint is the same. But if you say ORG:TEAM, then it only allows that team. Do these docs help explain what I mean? https://zero-to-jupyterhub.readthedocs.io/en/latest/administrator/authentication.html?highlight=allowed_organizations#github (I basically think `allowed_teams` doesn't exist and it should be `allowed_organzations`. The ORG:TEAM declaration is what narrows the scope to just a team.)

The `read:org` scope means that it shouldn't matter if they are private.

Reply:
Here -> there's no code matching `allowed_teams` in the z2jh helm chart: https://github.com/jupyterhub/zero-to-jupyterhub-k8s/search?q=allowed_teams

But there is `allowed_organizations`: https://github.com/jupyterhub/zero-to-jupyterhub-k8s/search?q=allowed_organizations

Reply:
`allowed_teams` is only a key for the kubespawner/profile list hack we have so members from different teams can see different images/machines: https://github.com/2i2c-org/infrastructure/blob/fe3375645d958085e8a927dbfcd9acedc97e750d/docs/howto/configure/auth-management.md#restricting-user-profiles-based-on-github-team-membership

But there is no key to the `GitHubOAuthenticator` called `allowed_teams` - only `allowed_organizations`

Reply:
Wow, I have no idea how I never realized this. Maybe I got confused by https://infrastructure.2i2c.org/en/latest/howto/configure/auth-management.html?highlight=github%20teams#restricting-user-profiles-based-on-github-team-membership? Pff, thank you! I'm changing that back now.

I'm now confused about why you were able to login? Was that because it was actually allowing everyone cause I set that wrong?

Reply:
Possibly - let's revisit after the meeting